.gitlab-ci.yml

```diff
@@ -313,13 +313,19 @@ deploy-ue1-tag:
   tags:
     - linux-docker
```

Comment:
since there is no only statement, it can be used to deploy every pull request, right?